computer.js

```diff
@@ -0,0 +1,124 @@
+var move = [];
+var numDoughnutsInTower = [NUM_DOUGHNUT, 0, 0];
+var cnt = 0;
+var HEIGHT = 250;
+
+
+function computerDet(doughnut, below, end)
+{
```

Comment:
Please be consistent with the coding style in the rest of the project. `{` should be in the same line as function definition.